queries/http/injections.scm

```diff
@@ -26,3 +26,15 @@
   (_
     (script) @injection.content
     (#offset! @injection.content 0 2 0 -2)))
+
+; post-request scripts for requests without body
+((request
+  !body
```

Comment:
I don't think that actually works, though...

Reply:
It is there to match `script` tag in cases like:
```http
GET localhost:9999

# @lang=lua
> {%
-- comment
%}
```
![IMG_1337](https://github.com/user-attachments/assets/1d1177ec-8037-4cb6-b245-3cb0f635ec20)

comment `# @lang=lua` should not close the `request` node to be used as body wrapper like `#@_RES … # @_END`. But we need “script node with variable declaration comment on top of it” query.
So that is what this query does; match `script` node inside `res_handler_script` that comes right after the `request` node ending with `comment` node with `lang` variable declaration.

Reply:
I mean the `!body` negated node -- I don't think that's supported in Neovim. (At least I've never seen it used in any query here.)

Reply:
Didn’t know about that, thank you for pointing out.
I’ll refactor the parser to simplify the queries.

Will this query be fine to be merged?

```scheme
; Script (default to javascript)
((#set! injection.language "javascript")
  (comment
    name: (_) @_name
    (#eq? @_name "lang")
    value: (_) @injection.language)?
  .
  (_
    (script) @injection.content
    (#offset! @injection.content 0 2 0 -2)))
```